app/models/organization.rb

```diff
@@ -105,17 +105,8 @@ def site_name
   # ask for help in this organization
   #
   # Warning: this method does not strictly check user's permission
-  def ask_for_help_enabled?(user = nil)
-    report_issue_enabled? || community_link.present? || can_create_discussions?(user)
-  end
-
-  # Tells if the given user can
-  # create discussion in this organization
-  #
-  # This is true only when this organization has a forum and the user
-  # has the discusser pseudo-permission
-  def can_create_discussions?(user = nil)
-    forum_enabled? && (!user || user.discusser_of?(self))
```

Comment:
This line was odd. It made nil users able to create_discussions :eye:

Reply:
Weird. It looks like it was on purpose, but I agree that this has little sense